library/styleguideComponents/Message/index.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import styled from 'styled-components';
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+import ButtonIcon from '../../../src/components/ButtonIcon';
+import RenderIf from '../../../src/components/RenderIf';
+import Notification from '../../../src/components/Notification';
+import Close from './icons/close';
+import MessageIcon from './messageIcon';
+import { hideMessage } from '../../../src/redux/actions/message';
+import './styles.css';
+
+function Message(props) {
+    // eslint-disable-next-line no-shadow
+    const { className, style, message, variant, hideMessage } = props;
+
+    const StyledNotification = styled(Notification)`
+        position: absolute;
+        z-index: 999999;
+    `;
+
+    function getClassName() {
+        return classnames(
+            'react-rainbow-message_container',
+            {
+                'react-rainbow-message_container--shown': message,
+            },
+            `react-rainbow-message_container-${variant}`,
+            className,
+        );
+    }
+
+    return (
+        <>
+            <RenderIf isTrue={!!message && variant === 'error'}>
```

Comment:
in order to keep the separation of concerns we should not mix js with jsx, then put this in a const